R/expr__expr.R

```diff
@@ -1,27 +1,88 @@
 #' Polars Expressions
 #'
-#' Expressions are all the functions and methods that are applicable to a Polars
-#' DataFrame or LazyFrame. They can be split into the following categories
-#' (following the [Py-Polars classification](https://pola-rs.github.io/polars/py-polars/html/reference/expressions/)):
-#'  * Aggregate
-#'  * Binary
-#'  * Categorical
-#'  * Computation
-#'  * Functions
-#'  * List
-#'  * Meta
-#'  * Name
-#'  * String
-#'  * Struct
-#'  * Temporal
+#' Expressions are all the functions and methods that are applicable to
+#' a Polars [DataFrame][DataFrame_class] or [LazyFrame][LazyFrame_class] object.
+#' Some methods are under the sub-namespaces.
 #'
+#' @section Sub-namespaces:
+#'
+#' ## arr
```

Comment:
Making a subsection with `##` for each category is not very useful, you could make a list of bullet points instead. Same thing for the active bindings `columns`, `shape`, etc.

Reply:
Yes, you are right here, but there is already a long explanation of active bindings.
https://github.com/pola-rs/r-polars/blob/c2ef53b16b73bcf25d4af15a44f6fbfb0373374c/R/dataframe__frame.R#L20-L35

Therefore, I thought it would be better to align the format of sub-namespaces with that of active bindings.

Reply:
I get your point, it's just that this takes unnecessary space since it's one almost identical sentence for each category:

![image](https://github.com/pola-rs/r-polars/assets/52219252/586b7dec-d92a-4f3b-b456-f6ea07762073)

compared to:

![image](https://github.com/pola-rs/r-polars/assets/52219252/4e3ca179-3ba2-46a0-88b6-7a3cd84d2799)

But that's not a big deal, just a matter of preference I guess. Leave it or change it ;)